modules/ROOT/pages/amazon/amazon-sns-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,41 @@
+= Amazon SNS - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|4.3.1 |4.4.0
+|===
+
+== Changes in This Release
+
+* Added the ability to configure TLS.
+* Upgraded `aws-java-sdk` from 1.11.611 to 1.11.649.
+* Upgraded `aws-connector-commons` from 2.0.0 to 2.2.0.
+
+== Requirements
+
+Ensure your system meets the following requirements before you perform the upgrade:
+
+[%header%autowidth.spread]
+|===
+|Application/Service|Version
+|Mule |4.1.1 and later
+|Amazon Java SDK for RDS|v1.11.649
+|Java|1.8.0_x
+|Anypoint Studio|7.0 and later
+|===
+
+See the xref:release-notes::connector/amazon-sns-connector-release-notes-mule-4.adoc[Amazon SNS Connector Release Notes] for updated compatibility information for patch updates.
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, create a backup of your files, data, and configuration in case you need to restore to the previous version.
+
+== See Also
+https://help.mulesoft.com[MuleSoft Help Center]
```

Comment:
```suggestion
* https://forums.mulesoft.com[MuleSoft Forum]
```

Reply:
Actually, these should all be references to the Help Center, which I think is replacing the Forum. I will fix the others.

Reply:
Oops! Thanks for catching that, and for fixing the others.